src/views/add-general-partner-person.njk

```diff
@@ -143,26 +143,32 @@
           items: [
             {
               classes: "govuk-input--width-2",
+              label: i18n.addPartnerPersonPage.dateOfBirthDay,
               name: "day",
               value: day,
               attributes: {
-                required: true
+                required: true,
+                pattern: "[0-9]*"
```

Comment:
Pattern format is present for the limited partner person page and as that was done after this one, I assume it was missing here so added it.